src/core/issues-notes.js

```diff
@@ -266,12 +266,16 @@ function isLight(rgb) {
 function createLabelsGroup(labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
   const labelNames = labels.map(label => label.name);
-  const joinedNames = joinAnd(labelNames);
+  const joinedNames = document.createElement("div");
```

Comment:
This specific one looks okay as it doesn't contain any HTML and is not inserted through `${[...]}`. It's safe as-is.